cas-server-support-saml/src/main/java/org/jasig/cas/support/saml/SamlProtocolConstants.java

```diff
@@ -28,4 +28,13 @@
     /** Indicates the endpoint for saml validation. */
     String ENDPOINT_SAML_VALIDATE = "/samlValidate";
 
+    /** The SAML2 SSO post profile endpoint. */
+    String ENDPOINT_SAML2_SSO_PROFILE_POST = "/idp/profile/SAML2/POST/SSO";
```

Comment:
Why are these constants not located in the `SamlIdpConstants`? They are SAML IdP constants, aren't they?